ljepotaservis/ljepotaservis.Web/ClientApp/src/utils/authHeader.js

```diff
@@ -1,9 +1,9 @@
-export function authHeader() {
+export const authHeader = () => {
   let user = JSON.parse(localStorage.getItem("user"));
```

Comment:
I probasbly missed this in the last review. Local storage is a colleciton of key value pairs. So you cannot getItem and expect an object, you will get an value. So maybe save UserToken or Token in local storage and when you do .getItem you will get the value of the token.

Reply:
The value on localstorage is a stringified (serialized) user object, after I get it JSON.parse converts it to an object so I think this should be ok.

Reply:
Okey that sounds good. It seemed fishy at first glance.